src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,344 @@
+                            - Math.log(1 / Math.pow(Number(numberNow), x))
+                    )
+                )
+            )
+
+            // Obtain the color of the circle
+            const combinedColor = this.primeFactors(
+                Number(numberNow),
+                Number(this.countPrime)
+            )
+
+            this.sketch.colorMode(this.sketch.HSB)
+            this.sketch.fill(combinedColor, 100, bright)
+            this.sketch.ellipse(this.X, this.Y, radius, radius)
+
+            // Change brightness regarding
+            //  the difference between 1/(number)^n and 1/(number)^(n-1)
+            bright =
+                (this.changeColor(diff, bright) * this.brightAdjust) / 100
+
+            radius -= 1
+        }
+    }
+
+    showCircleLabel(numberNow: number) {
+        this.sketch.fill('white')
+        this.sketch.text(
+            '1/'.concat(String(numberNow)).concat('^n'),
+            this.X,
+            this.Y
+        )
+    }
+
+    changePosition() {
+        this.X += 100
+        if (this.X >= this.canvasSizeX) {
+            this.X = this.canvasSizeX / this.n + 30
+            this.Y += 100
+        }
+    }
+
+    drawBox() {
+        //Create a white background for the description box
+        this.subG = this.sketch.createGraphics(this.boxSizeX, this.boxSizeY)
+        this.subG.colorMode(this.sketch.HSB)
+        this.subG.noStroke()
+        this.subG.fill(0, 0, 100)
+        this.subG.rect(0, 0, this.boxSizeX, this.boxSizeY)
+        this.subG.fill('black')
+        let tmpX = 0
+        let tmpY = 0
+
+        //Show the color of every prime number
+        for (let i = 0; i < this.primeNum.length; i++) {
+            this.subG.fill('black')
+            this.subG.text(String(this.primeNum[i]), 10 + tmpX, 15 + tmpY)
+            this.subG.fill(colorMap.get(this.primeNum[i]), 100, 100)
+
+            this.subG.ellipse(35 + tmpX, 15 + tmpY, 10, 10)
+            tmpX += 50
+            if (tmpX >= this.canvasSizeX) {
+                tmpX = 0
+                tmpY += 30
+            }
+        }
+        this.sketch.image(this.subG, 0, 700)
+    }
+
+    undrawBox() {
+        if (this.subG) {
+            //Draw a new black background to cover the description box
+            this.subG.fill('black')
+            this.subG.rect(0, 0, this.boxSizeX, this.boxSizeY)
+            this.sketch.image(this.subG, 0, 700)
+        }
+    }
+
+    //change the brightness of the circle
+    changeColor(difference: number, now: number) {
+        if ((now + difference) % 100 < now) {
+            return 100
+        }
+        return (now + difference) % 100
+    }
+
+    isPrime(n: number) {
+        for (let i = 2; i <= Math.sqrt(n); i++) {
+            if (n % i === 0) {
+                return false
+            }
+        }
+        return n > 1
+    }
+
+    //return a number which represents the color
+    primeFactors(n: number, totalPrime: number) {
```

Comment:
`isPrime()` and `primeFactors()` should use `sequence.getFactors()`

Reply:
done; you can take a look (repeats code from PR #230 (isPrime) but this is temporary I suppose)

Reply:
Seems OK now.